data/npclib/npc.lua

```diff
@@ -0,0 +1,108 @@
+-- Including the Advanced NPC System
+dofile('data/npclib/npc_system/npc_system.lua')
+dofile('data/npclib/npc_system/custom_modules.lua')
+
+isPlayerPremiumCallback = Player.isPremium
+
+-- Function called with by the function "Npc:sayWithDelay"
+local sayFunction = function(npc, text, type, e, pcid)
+	local npc = Npc(npc)
```

Comment:
This npc that comes as argument of the function is a string? If so can we call it npcName or similar?

Reply:
This function is called here: https://github.com/opentibiabr/canary/blob/c07bef1a570ab6914535aac2b3cf0d676ef49e36/data/npclib/npc.lua#L101